src/App.js

```diff
@@ -209,15 +201,15 @@ class App extends Component {
                         )
                     }
                 </svg>
-                {<button onClick={this.addText}>Add text</button>}
+                <button onClick={this.addText}>Add text</button>
                 <ImageUploader
                     onImageUpload={this.onImageUpload}
                 />
                 { activeElementIndex !== null ? <div>
                     <button onClick={this.removeElement}> Remove Circle </button>
                     <button onClick={() => this.moveFront(activeElementIndex)}> front </button>
                     <button onClick={() => this.moveBack(activeElementIndex)}> back </button>
-                    <input type='text'  onChange={this.editOnchange(activeElementIndex)} value={elements[activeElementIndex].props.children}/> <button onClick={this.updateText(activeElementIndex)}>Update</button>
+                    <input type='text'  onChange={this.editOnchange(activeElementIndex)} value={elements[activeElementIndex].props.children}/> 
```

Comment:
Seems good to me but we should show an Input box only if the selected element is a text. For e.g if you select an Image the input box will show which doesn't make any sense.

My suggestion will be having a type property on an element so we can check if the selected element is of which type.